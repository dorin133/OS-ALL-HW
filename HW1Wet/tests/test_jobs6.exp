smash> smash> smash: got ctrl-Z
smash: process 111004 was stopped
smash> [1] sleep 100& : 111003 2 secs
[2] sleep 200 : 111004 1 secs (stopped)
smash> smash> smash: got ctrl-Z
smash: process 111006 was stopped
smash> [1] sleep 100& : 111003 4 secs
[2] sleep 200 : 111004 3 secs (stopped)
[3] sleep 300& : 111005 2 secs
[4] sleep 400 : 111006 1 secs (stopped)
smash> smash: sending SIGKILL signal to 4 jobs:
111003: sleep 100&
111004: sleep 200
111005: sleep 300&
111006: sleep 400
