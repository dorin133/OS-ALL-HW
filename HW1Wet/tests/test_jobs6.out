smash> smash> smash: got ctrl-Z
smash: process 111005 was stopped
smash> [1] sleep 100& : 111003 2 secs
[2] sleep 200 : 111005 1 secs (stopped)
smash> smash> smash: got ctrl-Z
smash: process 111010 was stopped
smash> [1] sleep 100& : 111003 4 secs
[2] sleep 200 : 111005 3 secs (stopped)
[3] sleep 300& : 111009 2 secs
[4] sleep 400 : 111010 1 secs (stopped)
smash> smash: sending SIGKILL signal to 4 jobs:
111003: sleep 100&
111005: sleep 200
111009: sleep 300&
111010: sleep 400
